### PR TITLE
Fixed ODE errors

### LINKED ISSUE #7: The `spawner_demo.wbt` world displays ODE errors
I am getting the following error when running the world in Webots debug mode:
```
ODE Message 2: mass must be > 0 in dMassCheck() [ode/src/mass.cpp:48]
```

This doesn't affect the simulation, but it might be pointing to a Webots bug.

I managed to isolate the bug this far:
```
#VRML_SIM R2021b utf8
WorldInfo { }
Viewpoint { }
Robot {
  children [
    Solid {
      children [
        DEF FIXINGS_BO Transform {
          children [
            Shape {
              geometry Mesh {
                url "docking_element.obj"
              }
            }
          ]
        }
      ]
      boundingObject USE FIXINGS_BO
      physics Physics {
        density -1
        mass 2
        inertiaMatrix [
          1 1 1
          0 0 0
        ]
        centerOfMass [
          0 0 0
        ]
      }
    }
  ]
  boundingObject Shape {
    geometry Sphere {}
  }
  physics Physics {}
}
```
(the [docking_element.obj](https://github.com/cyberbotics/webots-projects/blob/master/projects/seronet-spawner/protos/meshes/docking_element.obj) needs to be present next to the world file)

### PR DESCRIPTION
**Pull request by mael25**
Should fix #7

### CODE REVIEW

**Comment by mael25** [comment]
I couldn't reproduce the error message in #7, but replaced the `boundingObject` of the _docking_element.obj_  `Mesh` by primitives, which should fix the issue.

I took also the opportunity to set a `density` instead of a `mass` for the `ConveyorStation`, so that it adapts according to its size.

**Comment by mael25** [comment]
If someone gets the warning described in #7, could they confirm that this solves the issue?

File: projects/seronet-spawner/protos/ConveyorStationPlatform.proto [hunk 1]
```diff
@@ -368,6 +353,47 @@ PROTO ConveyorStationPlatform [
...
+        children [
+          Shape {
+            appearance PBRAppearance {
+              baseColor 0.862 0.835 0.835
+              roughness 1
+              metalness 0
+            }
+              geometry Mesh {
+                url "meshes/docking_element.obj"
+              }
```
Comment (omichel) [inline projects/seronet-spawner/protos/ConveyorStationPlatform.proto]: Indentation looks wrong here:
```suggestion
            geometry Mesh {
              url "meshes/docking_element.obj"
            }
```

**Review by omichel — CHANGES_REQUESTED** [review]
I would revert the change with the mass/density as it now makes it impossible to change the mass of the ConveyorStation without changing its size...

**Comment by mael25** [comment]
The mass of the children of the `ConveyorStation` is now such that the total mass corresponds exactly to the one defined by the user.

**Review by omichel — APPROVED** [review]
Yes, it looks way better like that.

**Pull request merged**